src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/CLITest.java

```diff
@@ -158,11 +190,21 @@ public void interrupt() throws Exception {
         SSHD.get().setPort(0);
         File privkey = tmp.newFile("id_rsa");
         FileUtils.copyURLToFile(CLITest.class.getResource("id_rsa"), privkey);
-        User.get("admin").addProperty(new UserPropertyImpl(IOUtils.toString(CLITest.class.getResource("id_rsa.pub"))));
+        User.getById("admin", true).addProperty(new UserPropertyImpl(IOUtils.toString(CLITest.class.getResource(
+                "id_rsa.pub"),UTF_8)));
         FreeStyleProject p = r.createFreeStyleProject("p");
-        p.getBuildersList().add(new SleepBuilder(TimeUnit.MINUTES.toMillis(5)));
+        p.getBuildersList().add(new SleepBuilder(TimeUnit.MINUTES.toMillis(2)));
```

Comment:
I've changed the sleep to 120 seconds to avoid reach the 180 seconds test timeout.